docusaurus/docs/guides/discord/discord-webhooks.md

```diff
@@ -21,3 +21,7 @@ send discord messages with docker compose:
 - DISCORD_WEBHOOK_URL=https://discord.com/api/webhooks/1234567890/abcde
 - DISCORD_PRE_UPDATE_BOOT_MESSAGE=Server is updating...
 ```
+
+:::tip
+You can mention people in the messages by adding `<@user_id>` in the message!
+:::
```

Comment:
Not sure if it's a good idea, but couldn't we give the users the ability to just "@someone" directly and then parse it in the discord.sh to replace it with "<@someone>"? I think that would be more user friendly

Reply:
Sadly this is not the way discord webhooks work. You have to use the user_id, we cannot parse the @someone to a user_id

Reply:
Ah, I think I misunderstood it. Sorry my bad. I guess I was too tired yesterday.